src/Caching/Hybrid/src/Runtime/IsExternalInit.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+
+namespace System.Runtime.CompilerServices;
+
+#if !NET5_0_OR_GREATER
+[EditorBrowsable(EditorBrowsableState.Never)]
+internal static class IsExternalInit { } // for "init" support on down-level TFMs
```

Comment:
This already exists https://github.com/dotnet/aspnetcore/blob/79ef5e329b1e31c3775a1977798253cc8f7da6cc/src/Shared/IsExternalInit.cs#L7
Reference the shared file